js/contact-form.js

```diff
@@ -15,3 +15,34 @@ submit.addEventListener('click', (event) => {
     event.preventDefault();
   }
 }, false);
+
+
+//Form input to localStorage
+
+const form = document.getElementById('#form');
+form.addEventListener('submit', () => {
```

Comment:
- [ ] I kindly suggest that you wrap your functions for saving form data to local storage in an input event listener because your data object in local storage should be updated every time a user changes the content of any input field in your form.
- [ ] Please also ensure your saving data to localStorage functionality and showing it back to the user on reload is working as currently your data is not being saved to local storage at all
![image](https://user-images.githubusercontent.com/52934128/175774387-0e3ede0e-fb74-4583-b9a1-25468110a40c.png)


## EXAMPLE
form.addEventListener('input', () => {
  const data = {
    fullName: fullName.value,
    email: email.value,
    message: message.value,
  };

  localStorage.setItem('dataform', JSON.stringify(data));
});